js-guide.html

```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Newbie Guide: JavaScript</title>
+  </head>
+  <body>
+    <h1>Newbie Guide: JavaScript</h1>
+    <h2>Brief history of language</h2>
+    <p>JavaScript is an interpreted programming language originally created in 1995 by Brendan Eich. It is known as the language of the web, but it can also be run outside of a web browser.</p>
+
+    <h2>Where is Javascript used? What can I program with it?</h2>
+    <ul>
+      <li>Browser / client-side (websites UI, single-page-application frameworks)</li>
+      <li>Server-side / non-browser runtimes (NodeJS)</li>
+      <li>Additional cross-platform applications (Electron, IoT, anything else)</li>
+    </ul>
+
+    <h2>Key concepts &amp; best practices w/ Javascript</h2>
+    <ul>
+      <li>DOM (Document Object Model)</li>
+      <li>Primitive types (explain these briefly?)</li>
+      <li>Prototypical Inheritance</li>
+      <li>Closure</li>
+      <li>Functions as first-class citizens</li>
+      <li>ES6/2015 and onward</li>
+      <li>Module Systems (brief explanation of what these are)</li>
+    </ul>
+
```

Comment:
I'm wondering if it may still be useful to give a very broad overview of the JS Ecosystem and open-source... Brief intro of NPM modules, task runners like Gulp / Grunt, ES6, anything else? Just so people are aware of *what* those are.